src/ext/java/org/opentripplanner/ext/transmodelapi/mapping/PreferencesMapper.java

```diff
@@ -32,8 +32,8 @@ static void mapPreferences(
       }
     );
 
-    if (GqlUtil.hasArgument(environment, "maxAccessEgressDurationForMode")) {
```

Comment:
Here is an example of a test for this kind of mapper from the GTFS GraphQL API: https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/ext-test/java/org/opentripplanner/ext/legacygraphqlapi/mapping/RouteRequestMapperTest.java

Reply:
I created unit tests